CHANGELOG.md

```diff
@@ -1,5 +1,74 @@
+# 0.7.1 - Bugfix Release
```

Comment:
Same applies here regarding rebasing on your PR branch. You haven't made this changes, have you? Therefore they should not be part of your PR. 🤷

Reply:
Agreed, like above